api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-list-educationalactivities.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "List educationalActivities"
+description: "Get the educationalActivities from the educationalActivities navigation property."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List educationalActivities
+
+Namespace: Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models
+
+Get the educationalActivities from the educationalActivities navigation property.
```

Comment:
Should it be list or get?

Reply:
Great question. I would vote for "get"; it's unambiguously a verb, so there's less chance of forming an awkward description.
There are nav properties that start with "list". For example, the site entity has a nav property:
<NavigationProperty Name="lists" Type="Collection(graph.list)" ContainsTarget="true"/>